scripts/check_changeset.ts

```diff
@@ -37,12 +37,17 @@ const bumpRank: Record<string, number> = {
 // numerical rank, bump text, package name.
  */
 function getHighestBump(changesetPackages: Record<string, string>) {
+  const firebasePkgJson = require(resolve(
+    root,
+    'packages/firebase/package.json'
+  ));
   let highestBump = bumpRank.patch;
   let highestBumpText = 'patch';
   let bumpPackage = '';
   for (const pkgName of Object.keys(changesetPackages)) {
     if (
       pkgName !== 'firebase' &&
+      Object.keys(firebasePkgJson.dependencies).includes(pkgName) &&
```

Comment:
```suggestion
      pkgName in firebasePkgJson.dependencies &&
```

optional nit: shorter and more performant